yt_dlp/extractor/litv.py

```diff
@@ -113,7 +113,7 @@ def _real_extract(self, url):
             entry_protocol='m3u8_native', m3u8_id='hls')
         for a_format in formats:
             # LiTV HLS segments doesn't like compressions
-            a_format.setdefault('http_headers', {})['Youtubedl-no-compression'] = True
```

Comment:
+1. Surely `identity` was standardised when this was written? Someone hadn't read the specs.